minidump-common/src/errors/macos.rs

```diff
@@ -339,18 +339,21 @@ pub enum ExceptionCodeMacGuardType {
     GUARD_TYPE_USER = 3,
     GUARD_TYPE_VN = 4,
     GUARD_TYPE_VIRT_MEMORY = 5,
+    GUARD_TYPE_REJECTED_SC = 6,
 }
 
 /// Mac/iOS Mach port guard exception flavors
 ///
 /// See the [osfmk/mach/port.h][header] header in Apple's kernel sources
 ///
-/// [header]: https://github.com/apple/darwin-xnu/blob/main/osfmk/mach/port.h
+/// [header]: https://github.com/apple-oss-distributions/xnu/blob/main/osfmk/mach/port.h
 #[derive(Copy, Clone, PartialEq, Eq, Debug, FromPrimitive)]
 pub enum ExceptionCodeMacGuardMachPortFlavor {
-    GUARD_EXC_DESTROY = 0x00000001,
-    GUARD_EXC_MOD_REFS = 0x00000002,
-    GUARD_EXC_SET_CONTEXT = 0x00000004,
+    GUARD_EXC_DESTROY = 1,
+    GUARD_EXC_MOD_REFS = 2,
+    GUARD_EXC_INVALID_OPTIONS = 3,
+    GUARD_EXC_SET_CONTEXT = 4,
+    GUARD_EXC_THREAD_SET_STATE = 5,

```

Comment:
any reason for the change in notation?

Reply:
That's how they're written in Apple's headers now, and I like to stick to the same notation so that people grep'ing stuff around can find their way in the original sources.